app/controllers/feedback_controller.rb

```diff
@@ -9,6 +9,7 @@ def admin
 
   def new
     @feedback = Feedback.new
+    session[:subject] = params[:subject] || "Dispatch"
```

Comment:
I assumed that concept around subject was to refer to one piece of the Caseflow system, that's why 'Dispatch' was put, as one subsystem.